pkgs/development/libraries/science/math/nccl/tests.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, which
+, cudaPackages
+, mpiSupport ? false
+, mpi
+}:
+
+assert mpiSupport -> mpi != null;
+
+with cudaPackages;
+
+cudaPackages.backendStdenv.mkDerivation rec {
+
+  pname = "nccl-tests";
+  version = "2.13.6";
```

Comment:
```suggestion
  version = "2.13.6";
  strictDeps = true;
```

`strictDeps` should be used for CUDA packages to enforce run/build time dependencies.